packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -47,6 +48,8 @@ export class AttestationService {
   private readonly logger: ILogger;
 
   private nextAttesterDuties: Map<Slot, IAttesterDuty[]> = new Map();
+  private aggregatorTimeout?: NodeJS.Timeout;
```

Comment:
I'm afraid we can have multiple aggreagtor timeouts at the same time.
Might be better to wrap setting timeout into function with abort signal.
You can look at examples here:
https://ckeditor.com/blog/Aborting-a-signal-how-to-cancel-an-asynchronous-task-in-JavaScript/